include/seqan3/alignment/pairwise/execution/alignment_executor_two_way.hpp

```diff
@@ -166,9 +170,9 @@ class alignment_executor_two_way
         chunked_resource_it = chunked_resource.begin();
 
         if constexpr (std::same_as<execution_handler_t, execution_handler_parallel>)
-            init_buffer(std::ranges::distance(resrc));
+            init_buffer(std::ranges::distance(resrc), buffer_value);
```

Comment:
You take the parameter by copy so shouldn't you move here, and take the value by copy in the `init_buffer` too?

Reply:
Basically, it would be the same. So I would change it at this place. But in a more recent PR (not yet published) I'll remove this entirely. So I would like to keept it as is.

Reply:
It is really hard to review something and understand the rationale if "everything is changed in the next PR anyway". I know we asked you to split your PRs in smaller ones a while ago but I really have a hard time to keep track of what you are trying to do and that way I cannot really review the changes?

Reply:
In a week you can progress a lot with a PR which is a building block for more changes. This happens and I think a "I found another solution in the meantime, but I would like to get the changes that were made in here merged, s.t. the review work was not completely in vain" a plausible reason to defer this. I think rene is trust worthy enough if he promises them to do later.